src/api/controllers/db-controller.ts

```diff
@@ -730,6 +685,54 @@ function parseDbTxTypeMetadata(dbTx: DbTx | DbMempoolTx): TransactionMetadata {
   }
 }
 
+export function parseContractCallMetadata(tx: BaseTx): ContractCallTransactionMetadata {
```

Comment:
I think the bit of refactoring around this area creates some merge conflicts in the develop branch, @rafaelcr do you think the conflict look pretty easy to deal with later?

Reply:
No worries, since I'm moving to `develop` as the base branch now, I already made the conflict resolution (changes coming soon)